### DISCUSSION TITLE
Configuration Issue with nginx config and reverse proxy

### DISCUSSION THREAD

**Discussion by konsultaner** [verdaccio/verdaccio#2384] [Q&A] [answered]
I'm running a gitlab server that uses an nginx webserver. I added the following config to the nginx to make verdaccio run in the directory `/verdaccio/`

```
location ~ ^/verdaccio/(.*)$ {
proxy_pass https://127.0.0.1:4873/$1;
proxy_redirect off;
proxy_set_header X-Forwarded-Proto $scheme;
proxy_set_header X-Real-IP $remote_addr;
proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
proxy_set_header X-NginX-Proxy true;
proxy_set_header Host $host;
}
```

I took these settings from the [verdaccion docs](https://verdaccio.org/docs/reverse-proxy).

As you can see I proxy to a local port that servers verdaccio. the verdaccio config itself holds the these lines:

```yaml
listen: 'https://localhost:4873'
url_prefix: '/verdaccio/'
```

The[ verdaccio page](https://git.konsultaner.de/verdaccio) actually loads, but the assets are not loaded.

How do I config the assets path to work? For some reason the `-/static/` does not resolve correctly `static/` does but has no files. I'm confused.

**Comment by juanpicado** [comment, marked as answer]
https://verdaccio.org/blog#a-new-public-url-environment-variable

**Reply by konsultaner** [reply]
@juanpicado didn't read the guide carefully enough. I also found the `./storage` issue. But I'm now struggling with the fact that my webview does not show the private packages anymore eventhough I'm logged in. I use `.htpasswd` and my private packages are configured with:

```yaml
access: $authenticated
publish: $authenticated
proxy: npmjs
```

**Reply by juanpicado** [reply]
Use an absolute path if you can, https://verdaccio.org/blog/2021/04/14/verdaccio-5-migration-guide#after-upgrade-i-dont-see-packages-on-the-ui

**Reply by konsultaner** [reply]
I did and I see the public packages but not the private ones

**Reply by juanpicado** [reply]
🤔 ... are you migrating or is a new installation?

**Reply by konsultaner** [reply]
Found the issue. I had to relogin... sorry.
